InterfaceSwig/generated/tasmanian.f03

```diff
@@ -0,0 +1,3848 @@
+use, intrinsic :: ISO_C_BINDING
+import :: swigclasswrapper
+type(SwigClassWrapper), intent(inout) :: farg1
+type(SwigClassWrapper), intent(in) :: farg2
+end subroutine
+
+function swigc_makeEmpty() &
+bind(C, name="_wrap_makeEmpty") &
+result(fresult)
+use, intrinsic :: ISO_C_BINDING
+import :: swigclasswrapper
+type(SwigClassWrapper) :: fresult
+end function
+
+function swigc_makeGlobalGrid__SWIG_5(farg1, farg2, farg3, farg4, farg5) &
+bind(C, name="_wrap_makeGlobalGrid__SWIG_5") &
+result(fresult)
+use, intrinsic :: ISO_C_BINDING
+import :: swigclasswrapper
+integer(C_INT), intent(in) :: farg1
+integer(C_INT), intent(in) :: farg2
+integer(C_INT), intent(in) :: farg3
+integer(C_INT), intent(in) :: farg4
+integer(C_INT), intent(in) :: farg5
+type(SwigClassWrapper) :: fresult
+end function
+
+function swigc_makeSequenceGrid__SWIG_2(farg1, farg2, farg3, farg4, farg5) &
+bind(C, name="_wrap_makeSequenceGrid__SWIG_2") &
+result(fresult)
+use, intrinsic :: ISO_C_BINDING
+import :: swigclasswrapper
+integer(C_INT), intent(in) :: farg1
+integer(C_INT), intent(in) :: farg2
+integer(C_INT), intent(in) :: farg3
+integer(C_INT), intent(in) :: farg4
+integer(C_INT), intent(in) :: farg5
+type(SwigClassWrapper) :: fresult
+end function
+
+function swigc_makeLocalPolynomialGrid__SWIG_1(farg1, farg2, farg3, farg4, farg5) &
+bind(C, name="_wrap_makeLocalPolynomialGrid__SWIG_1") &
+result(fresult)
+use, intrinsic :: ISO_C_BINDING
+import :: swigclasswrapper
+integer(C_INT), intent(in) :: farg1
+integer(C_INT), intent(in) :: farg2
+integer(C_INT), intent(in) :: farg3
+integer(C_INT), intent(in) :: farg4
+integer(C_INT), intent(in) :: farg5
+type(SwigClassWrapper) :: fresult
+end function
+
+function swigc_makeLocalPolynomialGrid__SWIG_2(farg1, farg2, farg3, farg4) &
+bind(C, name="_wrap_makeLocalPolynomialGrid__SWIG_2") &
+result(fresult)
+use, intrinsic :: ISO_C_BINDING
+import :: swigclasswrapper
+integer(C_INT), intent(in) :: farg1
+integer(C_INT), intent(in) :: farg2
+integer(C_INT), intent(in) :: farg3
+integer(C_INT), intent(in) :: farg4
+type(SwigClassWrapper) :: fresult
+end function
+
+function swigc_makeLocalPolynomialGrid__SWIG_3(farg1, farg2, farg3) &
+bind(C, name="_wrap_makeLocalPolynomialGrid__SWIG_3") &
+result(fresult)
+use, intrinsic :: ISO_C_BINDING
+import :: swigclasswrapper
+integer(C_INT), intent(in) :: farg1
+integer(C_INT), intent(in) :: farg2
+integer(C_INT), intent(in) :: farg3
+type(SwigClassWrapper) :: fresult
+end function
+
+function swigc_makeWaveletGrid__SWIG_1(farg1, farg2, farg3, farg4) &
+bind(C, name="_wrap_makeWaveletGrid__SWIG_1") &
+result(fresult)
+use, intrinsic :: ISO_C_BINDING
+import :: swigclasswrapper
+integer(C_INT), intent(in) :: farg1
+integer(C_INT), intent(in) :: farg2
+integer(C_INT), intent(in) :: farg3
+integer(C_INT), intent(in) :: farg4
+type(SwigClassWrapper) :: fresult
+end function
+
+function swigc_makeWaveletGrid__SWIG_2(farg1, farg2, farg3) &
+bind(C, name="_wrap_ma
```

Comment:
I got to the point where I can generate the files locally, but I am getting errors when I try to compile.

Gfortran and I don't think this is a valid Fortran declaration.

Reply:
Github seems to be confusing the lines in a large file, the line with bad code is:
```
integer(C_INT), dimension(getPointsIndexes*) :: swig_result
```

Reply:
Aha, I know what's causing that: my unrestricted use of the `%apply` typemap to treat pointers as arrays isn't working correctly for the %typemap(out). I'll need to create a minimal test case and add to the `swig-fortran` issues page to track it.